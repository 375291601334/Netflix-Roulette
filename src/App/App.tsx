import React from 'react';
import './App.less';

export default function App() {
  return (
    <div>
      <h1>My React App!</h1>
    </div>
  );
}
