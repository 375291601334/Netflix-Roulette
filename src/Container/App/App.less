@import '../../Styles/globalStyles.less';
@import '../../Styles/variables.less';

.topLogo {
  position: absolute;
  top: 20px;
  left: 100px;

  .tablet({
    left: 50px;
  });

  .mobile({
    left: 30px;
  });
}
