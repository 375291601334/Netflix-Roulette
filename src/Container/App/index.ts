export { App, getServerSideProps } from './App';
