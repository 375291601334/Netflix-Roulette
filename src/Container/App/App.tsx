import React from 'react';
import { Header } from '../../Components/Header';
import { Footer } from '../../Components/Footer';
import { Logo } from '../../Components/Logo';
import { PageContent } from '../PageContent';
import { ErrorBoundary } from '../ErrorBoundary';
import './App.less';

export function App() {
  return (
    <>
      <Header />
      <ErrorBoundary>
        <PageContent key={'pageContent'} />
      </ErrorBoundary>
      <Footer>
        <Logo key={'logo'} />
      </Footer>
    </>
  );
}
