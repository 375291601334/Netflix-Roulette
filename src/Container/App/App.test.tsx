import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { App } from './App';

jest.mock('../../Pages', () => () => (<div>Main Page</div>));

describe('App', () => {
  test('render App', () => {
    const { baseElement } = render(<BrowserRouter><App /></BrowserRouter>);
    expect(baseElement).toMatchSnapshot();
  });
});
