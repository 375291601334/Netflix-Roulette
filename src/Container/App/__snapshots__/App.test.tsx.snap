// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render App 1`] = `
<body>
  <div>
    <div
      class="logo topLogo"
    >
      <span>
        netflix
      </span>
      roulette
    </div>
    <div>
      Main Page
    </div>
    <footer>
      <div
        class="logo undefined"
      >
        <span>
          netflix
        </span>
        roulette
      </div>
    </footer>
  </div>
</body>
`;
